.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
+  - npm run travis
```

Comment:
add scripts to run tests as well. `npm run test`.

Reply:
Hey @AmosWels,`npm run travis` basically runs `npm run test` but not in watch mode
`npm run test` runs the tests but keeps the build hanging
After the 10th-minute Travis treats it as an errored build.

Reply:
The build passes on the branch however travis looks at the default branch which id develop